dashboard/lib/logic/qualified_task.dart

```diff
@@ -4,59 +4,61 @@
 
 import 'package:flutter/foundation.dart';
 
-import '../model/task.pb.dart';
+import '../model/task_firestore.pb.dart';
 
-/// [Task.stageName] that maps to StageName enums.
-// TODO(chillers): Remove these and use StageName enum when available. https://github.com/flutter/cocoon/issues/441
 class StageName {
-  static const String cocoon = 'cocoon';
-  static const String legacyLuci = 'chromebot';
   static const String luci = 'luci';
-  static const String googleTest = 'google_internal';
   static const String dartInternal = 'dart-internal';
 }
 
+/// This is a side effect after deprecating `stage` field from Task model. A
+/// more generic way to detect dart internal targets may be preferred.
+const Set<String> dartInternalTasks = <String>{
+  'Linux engine_release_builder',
+  'Linux flutter_release',
+  'Linux flutter_release_builder',
+  'Linux packaging_release_builder',
+  'Mac engine_release_builder',
+  'Mac packaging_release_builder',
+  'Windows engine_release_builder',
+  'Windows packaging_release_builder',
+};
```

Comment:
@godofredoc Do we expect to have more dart internal tasks? I have been thinking about whether it deserves the effort to add a new `Task` field to identify these targets, similar to the old `stage`. If these targets are relatively steady, then a list like this may be enough. Or is there a pattern I can use here to determine whether a target is from dart-internal? like `*flutter_release*`/`*engine_release*`/`*packaging_release*`?